ragna/deploy/_api/database.py

```diff
@@ -211,7 +211,7 @@ def update_chat(session: Session, user: str, chat: schemas.Chat) -> None:
     orm_chat = _get_orm_chat(session, user=user, id=chat.id)
 
     orm_chat.prepared = chat.prepared
-    orm_chat.messages = [
+    orm_chat.messages = [  # type: ignore[assignment]
```

Comment:
mypy incorrectly infers `orm.Chat.messages` to be of type `orm.Message` rather than the correct `list[orm.Message]`. Thus, we just silence it here.